src/Display/ConsoleScreen.cs

```diff
@@ -12,8 +15,11 @@ public void Init(int width, int height)
         {
             // Avoid blinking by optimizing the buffering space in console
             // https://stackoverflow.com/questions/28490246/console-clear-blinking
```

Comment:
Thanks for testing on different platforms! (i didn't have much time to play) If that's Windows/Unix specific, then let's use preprocessor directives to keep this behavior in Windows and avoid in Unix. Like this https://stackoverflow.com/questions/30153797/c-sharp-preprocessor-differentiate-between-operating-systems